pages/profile/read/views/profile.jsx

```diff
@@ -16,224 +16,210 @@ class Profile extends React.Component {
       telephone,
       email,
       roles,
-      projects,
+      projects = [],
       establishments,
       id
     } = this.props.profile;
 
     const allowedActions = this.props.allowedActions;
-    const activeProjects = projects.filter(
-      ({ establishmentId, status }) =>
-        status === 'active' && establishmentId === estId
-    );
-    const estRoles = roles.filter(
-      ({ establishmentId }) => establishmentId === estId
-    );
+    const activeProjects = projects.filter(({ establishmentId, status }) => status === 'active' && establishmentId === estId);
+    const estRoles = roles.filter(({ establishmentId }) => establishmentId === estId);
+
+    const canSeeProjects = activeProjects && (isOwnProfile || allowedActions.includes('project.read.all'));
```

Comment:
`activeProjects` is an array, so will always resolve to true. There's no need to `&&` it here (or at least add a `.length`

Reply:
![](https://media.giphy.com/media/f3Mf34d21LKMg/giphy.gif)

Reply:
actual footage of me coding today